tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -23,276 +24,361 @@ def validator(value):
+        lambda value: value != "0" * number_of_zeros, lambda value: f"{value} is zero."
     )
 
 
 def isLargerThan(LowerBound):
     """Validate that value is larger than the given value."""
     return make_validator(
         lambda value: float(value) > LowerBound if value is not None else False,
-        lambda value: f'{value} is not larger than {LowerBound}.'
+        lambda value: f"{value} is not larger than {LowerBound}.",
     )
 
 
 def isSmallerThan(UpperBound):
     """Validate that value is smaller than the given value."""
     return make_validator(
         lambda value: value < UpperBound,
-        lambda value: f'{value} is not smaller than {UpperBound}.'
+        lambda value: f"{value} is not smaller than {UpperBound}.",
     )
 
+
 def isLargerThanOrEqualTo(LowerBound):
     """Validate that value is larger than the given value."""
     return make_validator(
         lambda value: value >= LowerBound,
-        lambda value: f'{value} is not larger than {LowerBound}.'
+        lambda value: f"{value} is not larger than {LowerBound}.",
     )
 
+
 def isSmallerThanOrEqualTo(UpperBound):
     """Validate that value is smaller than the given value."""
     return make_validator(
         lambda value: value <= UpperBound,
-        lambda value: f'{value} is not smaller than {UpperBound}.'
+        lambda value: f"{value} is not smaller than {UpperBound}.",
     )
 
+
 def isInLimits(LowerBound, UpperBound):
     """Validate that value is in a range including the limits."""
     return make_validator(
         lambda value: value >= LowerBound and value <= UpperBound,
-        lambda value: f'{value} is not larger or equal to {LowerBound} and smaller or equal to {UpperBound}.'
+        lambda value: f"{value} is not larger or equal to {LowerBound} and smaller or equal to {UpperBound}.",
     )
 
+
 # custom validators
 
+
 def dateMonthIsValid():
     """Validate that in a monthyear combination, the month is a valid month."""
     return make_validator(
         lambda value: int(str(value)[4:6]) in range(1, 13),
-        lambda value: f'{str(value)[4:6]} is not a valid month.'
+        lambda value: f"{str(value)[4:6]} is not a valid month.",
     )
 
+
 def olderThan(min_age):
     """Validate that value is larger than min_age."""
     return make_validator(
         lambda value: date.today().year - int(str(value)[:4]) > min_age,
-        lambda value: f'{date.today().year - int(str(value)[:4])} is not larger than {min_age}.'
+        lambda value: f"{date.today().year - int(str(value)[:4])} is not larger than {min_age}.",
     )
 
+
 def dateYearIsLargerThan(year):
     """Validate that in a monthyear combination, the year is larger than the given year."""
     return make_validator(
         lambda value: int(str(value)[:4]) > year,
-        lambda value: f'{str(value)[:4]} year must be larger than {year}.'
+        lambda value: f"{str(value)[:4]} year must be larger than {year}.",
     )
 
+
```

Comment:
I'm guessing all these minute changes came from our flake8 linter recommendations?

Reply:
Correct, and after our discussion yesterday, we can definitely parked them and revert the changes back until later